Sources/PrimerSDK/Classes/User Interface/TokenizationViewModels/ApplePayTokenizationViewModel.swift

```diff
@@ -397,6 +397,24 @@ extension ApplePayTokenizationViewModel: PKPaymentAuthorizationViewControllerDel
         didAuthorizePayment payment: PKPayment,
         handler completion: @escaping (PKPaymentAuthorizationResult) -> Void
     ) {
+#if targetEnvironment(simulator)
+        if payment.token.paymentData.count == 0 {
+            let err = PrimerError.invalidArchitecture(
+                description: "Apple Pay cannot return tokenization data in the simulator",
```

Comment:
Merchants don't know what "tokenization data" means here.
I'd put sth like: "Apple Pay does not work with Primer when used in the simulator due to a limitation from Apple Pay."

Reply:
Fixed